megaparsec-tests/tests/Text/Megaparsec/DebugSpec.hs

```diff
@@ -179,6 +186,7 @@ spec = do
 -- | Check that running the given parser on the input prints the expected
 -- string to the 'stderr'.
 shouldStderr ::
+  (HasCallStack) =>
```

Comment:
Why would this become necessary now?

Reply:
This is useful for searching failed tests in spec. Without this constraint error message refer to this function instead of appropriate test case